README.md

```diff
@@ -4,180 +4,54 @@
-```
+- *nix environment
+- Python 3.7+
 
-### Installing ID3
-
-A step by step series of examples that tell you how to get a development env running
-
-1. Clone the repo locally
+### Building
 
+Clone the repository locally and go to the root level of the repository. Run the following command:
 ```
-git clone https://github.com/matthewwong525/ID3_project_sickkids.git
+python3 setup.py sdist bdist_wheel
 ```
 
-2. Setup virtualenv and install dependencies
+### Installing
 
+Run the following command:
 ```
-cd ID3_project_sickkids
-virtualenv id3_env
-source id3_env/bin/activate
-pip3 install -r requirements.txt
+pip install dist/id3_variants_training-0.0.0-py3-none-any.whl
 ```
 
-3. Downloading 1000 genomes VCF files to digest (NEEDED FOR LOCAL API)
-
-Download this entire directory (http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/) into a `release` directory in the repo
-or run the following commands below
+### Usage
 
 ```
-wget -m ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/ -nd -P release -l 1
-```
-4. Test if everything works!
+usage: train-id3 [-h] [--diagram DIAGRAM] [--use-candig-apis]
+                 config_file model_file
 
-Run script below to see if everything works!
+positional arguments:
+  config_file        path to the config file that contains variant ranges in
+                     JSON format
+  model_file         path to output ID3 file
 
+optional arguments:
+  -h, --help         show this help message and exit
+  --diagram DIAGRAM  if provided, output diagram of tree to this file
+  --use-candig-apis  use remote API to access variant information rather than
+                     local VCF files
 ```
-python main.py
-```
-
-End with an example of getting some data out of the system or using it for a little demo
 
-## Updating `config.json`
-
-`config.json` has been already set up so you can run the classifier without any changes to this file. If you want to make some changes to your query, you can simply change some values within this file. Below are the explanations of the attributes within this file.
-
-```
-`variant_ranges` : The ranges of variants you want to inspect in the ID3 classifier. A variant range can contain more than one variant.
-`candig_server_url` : The url that points to the candig_server
-`candig_server_dataset_id` : The id of the dataset you want to query from. It is associated with the candig_server
-`user_mapping_path` : Path to the `.ped` file that maps individual ids to ancestries
-`chr_paths` : Path to the `.vcf` chromosome files from the 1000 genomes project
 ```
+usage: predict-id3 [-h] config_path model_file
 
-### Installing and starting candig_server
-
-Additionally, you can host the candig_server locally if you want to test the capabilities
+positional arguments:
+  config_path  path to the config file that contains vcf file paths
+  model_file   path to output ID3 file
 
+optional arguments:
+  -h, --help   show this help message and exit
 ```
```

Comment:
This is good; could you maybe add:

A specific case: e.g.
```
train-id3 test-cases/case3/config.json --diagram case3.png
predict-id3 test-cases/case3/test-config.json 
```

Explain what it does, and show the diagram that results?